streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -287,11 +287,22 @@
      * @param windows the specification of the aggregation window
      * @param <T>   the value type of the aggregated table
      */
-    <T, W extends Window> KTable<Windowed<K>, T> aggregateByKey(Aggregator<K, V, T> aggregator,
+    <T, W extends Window> KTable<Windowed<K>, T> aggregateByKey(T initValue,
```

Comment:
Possibly, we can make a clone by serializing the initialValue once and deserializing it for each key.